### PR TITLE
provide ISO8601 time format

### PR DESCRIPTION
**Pull request by matthewadams**

### CODE REVIEW

**Comment by matthewadams** [comment]
@mcollina no problem -- couldn't really find where they were located for these time format functions, but I'll have a looky-poo.

**Comment by matthewadams** [comment]
ok tests added

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino",
-  "version": "5.15.0",
+  "version": "5.16.0",
```
Comment (mcollina) [inline package.json]: please revert this

**Comment by matthewadams** [comment]
docs added

**Pull request merged**